source/manual/cdn.html.md

```diff
@@ -95,44 +74,28 @@ This will give you various debugging headers that may be useful:
 < Vary: Accept-Encoding, Accept-Encoding
 ```
 
-See the Varnish/Fastly docs for what these mean. Check out the Fastly
-[debugging guide](https://docs.fastly.com/guides/debugging/checking-cache#using-curl)
-for more details on testing.
+See the Varnish/Fastly docs for what these mean. Check out the Fastly [debugging guide](https://docs.fastly.com/guides/debugging/checking-cache#using-curl) for more details on testing.
 
 ## Fastly's IP ranges
 
-Fastly publish their cache node
-[IP address ranges as JSON from their API][fastly_ips]. We use these IP
-addresses in 2 places:
+Fastly publish their cache node [IP address ranges as JSON from their API][fastly_ips]. We use these IP addresses in 2 places:
 
-- Origin has [firewall rules][] in place so that only our office and Fastly
-  can connect.
-- Our [Fastly Varnish config][vcl_config] restricts HTTP purges to specific
-  IP addresses (otherwise anyone would be able to purge the cache).
+- Origin has [firewall rules][] in place so that only our office and Fastly can connect.
+- Our [Fastly Varnish config][vcl_config] restricts HTTP purges to specific IP addresses (otherwise anyone would be able to purge the cache).
 
-We have [a Jenkins job "Check CDN IP Ranges"][check-cdn-ip-ranges] which will
-start to fail if our Fastly IPs don't match the IPs returned from the Fastly
-API. If you see this alert, you can
-[let Reliability Engineering know][raise-with-re] and they will update our
-list of Fastly IPs to match the ones listed by Fastly.
+We have [a Jenkins job "Check CDN IP Ranges"][check-cdn-ip-ranges] which will start to fail if our Fastly IPs don't match the IPs returned from the Fastly API. If you see this alert, you can [let Reliability Engineering know][raise-with-re] and they will update our list of Fastly IPs to match the ones listed by Fastly.
 
-Updating the firewall rules in Carrenza with new Fastly IPs used to be done by
-committing the change to the govuk-provisioning repo and to then deploy the
-firewall through a jenkins job. This process is broken at the moment since the
-code base has diverged from the state of the firewall, while this is remedied
-we have to add the new rules manually, this is how to do it:
+Updating the firewall rules in Carrenza with new Fastly IPs used to be done by committing the change to the govuk-provisioning repo and to then deploy the firewall through a jenkins job. This process is broken at the moment since the code base has diverged from the state of the firewall, while this is remedied we have to add the new rules manually, this is how to do it:
```

Comment:
```suggestion
Updating the firewall rules in Carrenza with new Fastly IPs used to be done by committing the change to the govuk-provisioning repo and to then deploy the firewall through a jenkins job. This process is broken at the moment since the code base has diverged from the state of the firewall. While this is being remedied, we have to add the new rules manually; this is how to do it:
```

Reply:
Done ✅